src/__tests__/if-then-else-spec.js

```diff
@@ -55,3 +55,32 @@ test("ternary", t =>
   export function test(): i32 {
     return 1 ? 42 : 24;
   }`).then(outputIs(t, 42)));
+
+test("else if statement", t =>
+  compileAndRun(`
+  export function test(): i32 {
+    let x: i32 = 1;
+    let y: i32 = 0;
+    if (x == 0) {
+      y = 2;
+    } else if (x == 1) {
+      y = 4;
+    } else {
+      y = 1;
+    }
+    return y;
+  }`).then(outputIs(t, 4)));
+
+  test("else if statement no {}", t =>
+  compileAndRun(`
+  export function test(): i32 {
+    let x: i32 = 1;
+    let y: i32 = 0;
+    if (x == 0)
+      y = 2;
+    else if (x == 1)
+      y = 4;
+    else
+      y = 1;
+    return y;
```

Comment:
Thank you for the tests.

Please make `test` take an `i32` argument for `x` so that we could test all the branches, I want to make sure we cover all the scenarios.

something like this would work
```js
WebAssembly.instantiate(compile(src)).then(({ instance: { exports } }) => {
   t.is(exports.test(0), 2);
   t.is(exports.test(1), 4);
   t.is(exports.test(-1), 1);
});
```